core/src/main/java/feast/core/config/FeastProperties.java

```diff
@@ -103,8 +103,18 @@ LoggingProperties loggingProperties() {
     /* The active Apache Beam runner name. This name references one instance of the Runner class */
     private String activeRunner;
 
-    /* If true only one IngestionJob would be created per source with all subscribed stores in it */
-    private Boolean consolidateJobsPerSource = false;
+    /* Job Coordinator related properties */
+    private CoordinatorProperties coordinator;
+
+    @Getter
+    @Setter
+    public static class CoordinatorProperties {
+      /* If true only one IngestionJob would be created per source with all subscribed stores in it */
+      private Boolean consolidateJobsPerSource = false;
+
+      /* Labels to identify jobs managed by this job coordinator */
+      private Map<String, String> jobSelector = new HashMap<>();
```

Comment:
Would it be possible to consolidate these options in the Job Runner configuration? The relationship is 1:1 right?

Reply:
Not sure wdym. This selector is meant to be used by all runners that support labels/filters or have any state. And job runner has only `options` that is used to spawn new job and kinda platform specific.
I was also planning to add here feature-sets selector to specify which feature sets this instance is responsible for. So idea was to keep those configs together.